cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -640,6 +640,50 @@ impl ABIMachineSpec for X64ABIMachineSpec {
     ) -> SmallVec<[Self::I; 16]> {
         let mut insts = SmallVec::new();
 
+        // When a return_call within this function required more stack arguments than we have
+        // present, resize the incoming argument area of the frame to accommodate those arguments.
+        let incoming_args_diff = frame_layout.tail_args_size - frame_layout.incoming_args_size;
```

Comment:
We may need to add this to `offset_upward_to_caller_sp` in the unwind info below.